### ISSUE TITLE
Restrict secp*r1 curves

### ISSUE THREAD

**Issue by msporny** [w3c-ccg/lds-jws2020#16]
The following secp*r1 curves are supported currently:

> EC	P-256	ES256
> EC	P-384	ES384
> EC	P-521	ES512

The whole point with year-stamped cryptographic suites is to pick *one* option that makes sense for that year. I suggest P-384, since it seems to meet all national and enterprise standards across the globe.

**Comment by kdenhartog** [comment]
I'm good with that. Last I heard @troyronda may have a requirement for P-256/ES256 as well. Definitely don't think we need P-521/ES512 support here.

I like the idea of restricting on

**Comment by troyronda** [comment]
We are using this suite with P256 curves.

**Comment by OR13** [comment]
I agree we can probably drop P-521/ES512... I'm not sure its helpful to drop `EC P-256 ES256` and `EC P-384 ES384`... as I understand it, both are used widely in legacy systems... 

I would be in favor of Jws2021 having dropping to only support for `EC P-384 ES384` on the NIST Curve front.
